docker/puppetdb/Dockerfile

```diff
@@ -1,57 +1,32 @@
-FROM clojure:openjdk-8-lein AS builder
-RUN apt-get update && \
-    apt-get install -y --no-install-recommends make
-# Install only dependencies
-WORKDIR /app
-COPY project.clj /app/
-COPY resources/puppetlabs/puppetdb/bootstrap.cfg \
```

Comment:
Looks like `bootstrap.cfg` might have been missed?

```
20 Execution error (IllegalArgumentException) at sun.reflect.NativeConstructorAccessorImpl/newInstance0 (NativeConstructorAccessorImpl.java:-2).
20 Problem loading service '../../puppetlabs/puppetdb/bootstrap.cfg' from /etc/puppetlabs/puppetdb/bootstrap.cfg:1:
20 Invalid line in bootstrap config file:
20 
20 	../../puppetlabs/puppetdb/bootstrap.cfg
20 
20 All lines must be of the form: '<namespace>/<service-fn-name>'.
```

Reply:
hmm, we shouldn't need to include that separately since this is building a puppetdb package from source now (and it's working on non-windows), will hop onto the azure builder to investigate

Reply:
turns out `resources/ext/config/bootstrap.cfg` is a symlink point to `resources/puppetlabs/puppetdb/bootstrap.cfg`, which is why it was failing on windows. Updated the `COPY`s below to copy over `bootstrap.cfg` explicitly